polkadot/node/network/availability-recovery/src/lib.rs

```diff
@@ -1155,7 +538,11 @@ impl AvailabilityRecoverySubsystem {
 		req_receiver: IncomingRequestReceiver<request_v1::AvailableDataFetchingRequest>,
 		metrics: Metrics,
 	) -> Self {
-		Self { recovery_strategy: RecoveryStrategy::BypassAvailabilityStore, req_receiver, metrics }
+		Self {
+			recovery_strategy_kind: RecoveryStrategyKind::BypassAvailabilityStore,
```

Comment:
We don't seem to yet use multiple strategies.  We'd need to break down `with_chunks_if_pov_large` into two separate strategies for that.

Reply:
we do use multiple strategies. This is just a constructor for the `AvailabilityRecoverySubsystem` which I kept. The strategy is created later on in `handle_recover`